filament/src/PostProcessManager.cpp

```diff
@@ -2697,6 +2654,70 @@ void PostProcessManager::configureTemporalAntiAliasingMaterial(
     }
 }
 
+FMaterialInstance* PostProcessManager::configureColorGradingMaterial(
+        PostProcessMaterial& material, FColorGrading const* colorGrading,
+        ColorGradingConfig const& colorGradingConfig, VignetteOptions const& vignetteOptions,
+        uint32_t const width, uint32_t const height) noexcept {
+    FMaterial* const ma = material.getMaterial(mEngine);
+    bool dirty = false;
+
+    auto setConstantParameter =
+            [&dirty](FMaterial* const material, std::string_view const name, auto value) noexcept {
+        auto id = material->getSpecializationConstantId(name);
+        if (id.has_value()) {
+            if (material->setConstant(id.value(), value)) {
+                dirty = true;
+            }
+        }
+    };
+
+    setConstantParameter(ma, "isOneDimensional", colorGrading->isOneDimensional());
+    setConstantParameter(ma, "isLDR", colorGrading->isLDR());
+
+    if (dirty) {
+        ma->invalidate();
+        // TODO: call Material::compile(), we can't si that now because it works only
```

Comment:
"can't si" ?

Reply:
I copied this from the above comment, but fixed